radsat-sk/operation/message/protobuf/types/RFileTransfer.pb.h

```diff
@@ -9,87 +9,348 @@
 #error Regenerate this file with the current version of nanopb generator.
 #endif
 
+/* Enum definitions */
+typedef enum _ImagePacket_image_type_t {
+    ImagePacket_image_type_t_fullResolution = 0,
+    ImagePacket_image_type_t_halfResolution = 1,
+    ImagePacket_image_type_t_quarterResolution = 2,
+    ImagePacket_image_type_t_thumbnail = 3
+} ImagePacket_image_type_t;
+
 /* Struct definitions */
-typedef PB_BYTES_ARRAY_T(200) FileTransferPacket_data_t;
-typedef struct _FileTransferPacket {
-    int32_t packetType;
-    int32_t packetNumber;
-    FileTransferPacket_data_t data;
-} FileTransferPacket;
-
-typedef struct _FileTransferResponse {
-    int32_t packetType;
-    int32_t packetNumber;
-    int32_t response;
-} FileTransferResponse;
+typedef struct _AntennaTelemetry {
+    char dummy_field;
+} AntennaTelemetry;
+
+typedef struct _BatteryTelemetry {
+    char dummy_field;
+} BatteryTelemetry;
+
+typedef struct _EpsTelemetry {
+    char dummy_field;
```

Comment:
create TODO to finish struct

Reply:
These are based off of the `.proto` definitions